doc/source/guide_lenses.rst

```diff
@@ -0,0 +1,117 @@
+Guide to Lenses
+===============
+
+Overview
+--------
+
+A Lens can be either a zone-app or a service.
```

Comment:
Securing everything is an euphemism for securing nothing, or at best doing best effort security. 

Instead you should confidently and precisely secure what matters. Even in the light of an intruder within your premises. Zero Trust teaches us to practice security the way our Secret Service protects the President of United States. At any point, they need to know who the president is, where (s)he is at all times, and who has access to them at all times. Its about identifying your protect surface, and about lensing your concern, about creating focus.

Lenses are the minimum yet complete self-contained units that can be owned, reviewed, monitored, and enforced. They can either be applications that Araali protects, or services that are used by Araali protected clients. By lensing your concern you get to focus on one app at a time, or one service at a time, wherever your concern may lie.

Turning enforcement on a Araali protected app means that only authorized clients can use the services offered by the app (ingress). Should the app get compromised by an intruder, it doesn't pollute the environment or move laterally (egress), or even get access to services that are internal to the app (internal).

Turning enforcement on a service lens means that only approved apps from all Araali protected assets can ever access the service. If there is an intruder within the Araali protected environment, it doesn't get the same access to the service, as your own legitimate apps would. Service, by definition is a third party service that Araali does not have presence in. So the enforcement happens on the client side, making sure only the right identities get to access an external service from within your environment.

But before we get into enforcement, we have to first identity what lenses are worth spending time on. You can pin important lenses to your dashboard using our APIs, check the enforcement status, or turn enforcement on or off.